lib/helpers.js

```diff
@@ -1,69 +1,97 @@
-const authToken = require('./auth-token');
-const User = require('../models/user');
-const Driver = require('../models/driver');
+const authToken = require('./auth-token')
+const User = require('../models/user')
+const Driver = require('../models/driver')
+const safeEquals = require('safe-compare')
```

Comment:
is completely necessary add this module? or what is the advantage? 🤔

Reply:
@noelrock333  When comparing password hashes or security tokens, we should use a constant time algorithm to prevent timing attacks. This library is used for that, I can implement it myself if you don't want a library tho.

https://snyk.io/blog/node-js-timing-attack-ccc-ctf